cclib/parser/gaussianparser.py

```diff
@@ -1721,7 +1721,7 @@ def extract(self, inputfile, line):
 
                 colmNames = next(inputfile)
                 
-                if "Density Matrix:" in colmNames or "DENSITY MATRIX." in colmNames:
+                if "Density Matrix:" in colmNames or "DENSITY MATRIX." in colmNames or "Beta Molecular Orbital Coefficients" in colmNames:
```

Comment:
Can you do an `if any(name in colmNames for name in ["Density Matrix:", ...`? Sometimes when this happens we stick the variable with possible headers on the class instance in `before_parsing`. It gets cleaned up when attributes are filtered out and might make the parser a little faster since it doesn't need to reevaluate one list of strings every time the condition is checked. (See https://github.com/cclib/cclib/blob/49b0cd6056de292b5cb1eb4d66da96e540e94493/cclib/parser/qchemparser.py#L95.)